object-assign/object-assign.d.ts

```diff
@@ -4,6 +4,5 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare module "object-assign" {
-  function objectAssign(target: any, ...sources: any[]): any;
-  export = objectAssign;
+  export default function(target: any, ...sources: any[]): any;
```

Comment:
this is not correct. this package does not export a default. it is a CJS module, as per the documentation, see https://github.com/sindresorhus/object-assign#usage

```ts
const objectAssign = require('object-assign');
```

babel/system fudge this and make the default export the CJS module, but this is not accurate.